cmake/scripts/linux/ArchSetup.cmake

```diff
@@ -41,6 +41,6 @@ if(CMAKE_BUILD_TYPE STREQUAL Coverage)
   set(COVERAGE_EXCLUDES */test/* lib/* */lib/*)
 endif()
 
-if(ENABLE_MIR)
+if(ENABLE_MIR OR ENABLE_GBM)
```

Comment:
I understand that the inverted logic is ugly, but I am not sure how to fix it. How can we make it so that vdpau is disabled for mir and gbm but is optional for x11? Perhaps this can be cleaned up in a future PR.